components/Map.js

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+import ReactMapGL, { Marker } from "react-map-gl";
+export default function Map({ locations }) {
```

Comment:
as you are using functional components, you should use arrow functions.

const Map = ({locations} => { ... }

export default Map

(this is a good practice)

Reply:
ok i don't know that, we use Next since 1 month more or less,  and teacher doesn't explain much